src/lib.rs

```diff
@@ -39,6 +39,39 @@ pub fn parse_and_inline_modules(src_file: &Path) -> syn::File {
         .output
 }
 
+/// The string `syn_inline_mod_path`. This acts as a marker for inlined paths.
+pub static SYN_INLINE_MOD_PATH: &str = "syn_inline_mod_path";
```

Comment:
Does this need to be `pub` or can it be restricted to the crate?

Reply:
Doesn't need to be pub -- will change.